lib/checkleakautovar.cpp

```diff
@@ -52,6 +52,8 @@ static const CWE CWE415(415U);
 static const int NEW_ARRAY = -2;
 static const int NEW = -1;
 
+static const std::vector<std::pair<std::string, std::string>> var1_fail_conds = {{"==", "0"}, {"<", "0"}, {"==", "-1"}, {"<=", "-1"}};
```

Comment:
to me it feels like `var1_fail_conds` are the failure conditions. and `var2_fail_conds` seem to be success conditions. What do you think about renaming them to `alloc_failed_conds` and `alloc_success_conds`. I am not against replacing "alloc" with "var" if you would prefer that.

Reply:
👍